frontend/functions/create-status.ts

```diff
@@ -15,6 +14,7 @@ export interface pastConditionsProps{
     Temperature: number;
     StatusTime: string;
     IsReviewed?: boolean;
+    Patient_PatientId?: number;
     UserId?: number;
     Weight: number;
     Symptoms: string;
```

Comment:
isnt this exactly the same interface as `StatusDataProps` in` frontend/models/patient.ts`